### ISSUE TITLE
Error detected while processing function UltiSnips#TrackChange

### ISSUE THREAD

**Issue by SkewedZone** [SirVer/ultisnips#1300]
Whenever I'm in Insert Mode and try to write something in any file, I get a myriad of errors, such as:

```
Error detected while processing C:\Users\user\.vim\plugged\ultisnips\autoload\UltiSnips.vim

E319: Sorry, the command is not available in this version: py3 import vim

Error detected while processing C:\Users\user\.vim\plugged\ultisnips\autoload\UltiSnips.vim

E319: Sorry, the command is not available in this version: py3 from UltiSnips import UltiSnips_Manager

Error detected while processing function UltiSnips#TrackChange

E319: Sorry, the command is not available in this version: py3 UltiSnips_Manager._track_change()
```

The _vimrc file I configured can be found here: https://github.com/SkewedZone/VIMRC-File

-----

- **Operating System**:  Windows 10 Home
- **Vim Version**:  VIM - Vi IMproved 8.2 
MS-Windows 32-bit console version
Compiled by mool@tororo
Huge version without GUI. 
- **Python inside Vim**:  3.6.5

**Comment by SirVer** [comment]
> Sorry, the command is not available in this version

this indicates, that your vim does not have python3 support. You can check by starting it and running `:py3 import sys;print(sys.version)`. That should give the same warning.

**Comment by SkewedZone** [comment]
I finally managed to solve it after a day, and started out by doing `:echo has("python3")` which returned 0, so python3 wasn't indeed not supporting vim.
The workaround was, first making sure that both vim and python3 were either both 32bit or both 64 bit (my vim was 32 bit and python3 was 64 bit, so I reinstalled vim in the 64bit package) and then making sure that python3 version installed on the pc supported the vim version. 
Whenever I tried to insert anything, vim would detect an error saying that I required python38.dll, and I had python 3.6.5 installed on the pc, the fix was just installing a version of python 3.8, setting it up on the environment variables and it was finally fixed.
